site/blog/2018-07-16-tutorial-template/README.md

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
@RobGrootjen this is a **template** no? So should not be migrated ... Please pay attention 😉